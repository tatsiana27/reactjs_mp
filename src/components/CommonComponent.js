import React, { Component } from 'react';

export class CommonComponent extends Component {
    render() {
        return(
            <div className="title">
                netflixroulette
            </div>
        );
    }
}
