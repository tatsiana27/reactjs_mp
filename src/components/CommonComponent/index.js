import React, { Component } from 'react';

export class CommonComponent extends Component {
  render() {
    return(
      <div className="common_title">
        netflixroulette
      </div>
    );
  }
}
