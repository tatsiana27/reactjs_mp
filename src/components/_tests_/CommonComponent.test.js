import React from 'react';
import { CommonComponent } from '../CommonComponent';
import { shallow } from 'enzyme';

describe('CommonComponent ', () => {
    test('should render CommonComponent', () => {
        const wrapper = shallow(
            <CommonComponent/>

        );

        expect(wrapper).toMatchSnapshot();
    });
});