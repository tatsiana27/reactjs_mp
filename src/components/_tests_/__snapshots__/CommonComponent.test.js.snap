// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonComponent  should render CommonComponent 1`] = `
<div
  className="common_title"
>
  netflixroulette
</div>
`;
